lib/dotcom_web/live/trip_planner.ex

```diff
@@ -76,27 +76,25 @@ defmodule DotcomWeb.Live.TripPlanner do
       <div>
         <.results_summary class="mt-2 mb-6" changeset={@input_form.changeset} results={@results} />
         <div class={[
-          "flex flex-col gap-4 md:flex-row md:gap-7"
+          "flex flex-col-reverse gap-4 md:flex-row md:gap-7"
         ]}>
+          <.results
+            :if={Enum.count(@results.itinerary_groups) > 0 || @results.loading?}
+            class="md:max-w-[25rem] md:sticky md:top-4"
+            results={@results}
+          />
           <.live_component
             module={MbtaMetro.Live.Map}
             id="trip-planner-map"
             class={[
-              "md:order-last md:sticky md:top-4",
-              "h-64 md:h-[32rem] w-full",
-              @results.itinerary_group_selection == nil && "hidden md:block",
-              @results.itinerary_group_selection != nil && "block"
```

Comment:
I'm pretty sure we still want this logic, since we only want the map to show up if you're looking at a specific itinerary. (Unless that part of the design changed?)